planning/behavior_path_planner/docs/behavior_path_planner_drivable_area_design.md

```diff
@@ -104,22 +107,53 @@ Note that we only expand right bound of the rightmost lane and left bound of the
 
 #### Dynamic Expansion
 
-The drivable area can also be expanded dynamically by considering the ego vehicle footprint projected on each path point.
-This expansion can be summarized with the following steps:
+The drivable area can also be expanded dynamically based on a minimum width calculated from the path curvature and the ego vehicle's properties.
```

Comment:
Is the `dynamic` DA incorporates information from `static` DA? or `static` DA is not considered when generating the `dynamic` DA?